tests/python_package_test/test_dask.py

```diff
@@ -220,8 +386,62 @@ def test_regressor_local_predict(client, listen_port):
     s2 = dask_regressor.to_local().score(X, y)
 
     # Predictions and scores should be the same
-    assert_eq(p1, p2)
     assert_eq(s1, s2)
+    assert_eq(p1, p2)
```

Comment:
can you please move this line back up above `assert_eq(s1, s2)`, so it drops out of the diff?

Reply:
Addressed in d1d4abf